src/Cashout/AbstractOperationProcessor.php

```diff
@@ -138,4 +140,35 @@ public function hasSufficientFunds($amount, $vendor, $transfer = false)
             throw new WrongWalletBalance($vendor->getHipayId(), 'withdraw', $amount, $balance);
         }
     }
-}
\ No newline at end of file
+
+
+    protected function checkOperationVendorEnabled($vendor, $operation)
+    {
+        if($vendor === $this->operator){
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Put one space between the closing parenthesis and the opening curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1808)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Put one space between this "if" keyword and the opening parenthesis. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1808)